.github/workflows/builds.yml

```diff
@@ -17,7 +17,7 @@ env:
   DEFAULT_SV_BRANCH: master
   DEFAULT_SV_PR:
 
-  DEFAULT_CFLAGS: "-Wall -Wextra -Werror -Wno-unused-parameter -Wno-unused-function"
+  DEFAULT_CFLAGS: "-Wall -Wextra -Werror -Wno-unused-parameter -Wno-unused-function -Wimplicit-int-float-conversion"
```

Comment:
@jasonish where should I put this ?
This flag is not supported on the CentOS 8 build

Reply:
cc @ct0br0